### PR TITLE
add Experimenter.config and lengthen Session.userIP

### PR DESCRIPTION
**Pull request by mtbc**
# What this PR does

Adds a new map annotation property `Experimenter.config` for expanding permissions. Lengthens `Session.userIP` so that it can accommodate IPv6 addresses.

# Testing this PR

Try using `Experimenter.config`. For instance, as root,
```
bin/omero obj map-set Experimenter:3 config foo bar
bin/omero obj map-get Experimenter:3 config foo
```

Check if one can log in to OMERO from an IPv6 host if convenient, but note new coverage by `testSessionWithIP` at https://ci.openmicroscopy.org/job/OMERO-DEV-merge-integration-python/lastCompletedBuild/testReport/OmeroPy.test.integration.test_isession/TestISession/.

# Related reading

https://trello.com/c/t0nT7KYa/35-new-role and http://trac.openmicroscopy.org/ome/ticket/13254.

### CODE REVIEW

File: components/tools/OmeroPy/test/integration/test_isession.py [hunk 1]
```diff
@@ -288,7 +288,11 @@ def testManageMySessions(self):
         finally:
             c1.__del__()
 
-    def testSessionWithIP(self):
+    @pytest.mark.parametrize("client_ip", [
+        "127.0.0.1",
+        "2400:cb00:2048:1::6814:55",
+        "1234:5678:1234:5678:1234:5678:121.212.121.212"])
```
Comment (mtbc) [inline components/tools/OmeroPy/test/integration/test_isession.py]: for the curious reviewer, this imitates a IPv4-mapped IPv6 address from a dual-stack node

**Comment by joshmoore** [comment]
Changes all match my expectations.

**Comment by mtbc** [comment]
@jburel: Looks good to you? If so I'll remove from breaking.

**Comment by jburel** [comment]
looks good @mtbc

**Pull request merged**